README.md

```diff
@@ -68,3 +68,61 @@ function SaveButton() {
     return <Button>Hello World</Button>;
 }
 ```
+
+<br />
+
+## Linking and Unlinking for Local Packages
+
+At times it may be required to link an external pacakge to a local version of the admin ui for testing. To do this have 2 terminals open. One terminal should be in the root of your local folder of the admin ui package and the other should be in the root to the project you would like to use the local package in. VS code should also be open.
+
+Examples used here are admin and admin-ui.
+
+### Linking
+
+```
+In admin-ui terminal
+    1. npm build
+    2. npm link
+```
+
+```
+In admin terminal
+    3. npm build
```

Comment:
@NoahThomlison should this be npm install?

Reply:
Updated